wasm-engines/rust-code/bn128_add/Cargo.toml

```diff
@@ -4,8 +4,7 @@ version = "0.1.0"
 edition = "2018"
 
 [dependencies]
-ethereum-bn128 = { git = "https://github.com/ewasm/ethereum-bn128.rs", branch = "bn-alloc" }
-parity-bytes = { git = "https://github.com/paritytech/parity-common" }
```

Comment:
0.1.1 was a cleanup locking in every upstream version and it also got rid of parity-bytes. Much success.